src/dolphin/unwrap/_unwrap_3d.py

```diff
@@ -108,3 +118,29 @@ def _map_date_str_to_file(
     # None is special case for reference epoch
     date_str_to_file[first_date] = None
     return date_str_to_file
+
+
+def _create_conncomps_from_mask(
+    temporal_coherence_file: PathOrStr,
+    temporal_coherence_threshold: float,
+    unw_filenames: Sequence[PathOrStr],
+) -> list[Path]:
+    arr = io.load_gdal(temporal_coherence_file, masked=True)
+    # XX: for now, "connected component" files are
+    # - 0 for invalid, ignored pixels
+    # - 1 for the good ones
```

Comment:
So each connected component will have the same label, even if there are multiple disjoint components?

Maybe it'd be worthwhile to use something like [`scipy.ndimage.label()`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.label.html#scipy.ndimage.label) here to convert the binary mask into a set of unique connected component labels? Otherwise, it seems like it might be confusing for users that the definition of "connected components" is different for spurt than for the other unwrappers.

Reply:
oh yeah i do like that idea. I wasn't really a fan of just 0/1, but somehow forgot about that 👍